redirects/18.0.txt

```diff
@@ -23,3 +23,7 @@ applications/sales/sales/send_quotations/optional_products.rst applications/sale
 applications/sales/sales/send_quotations/orders_and_variants.rst applications/sales/sales/sales_quotations/orders_and_variants.rst               # sales/send_quotations/* --> sales/sales_quotations/*
 applications/sales/sales/send_quotations/pdf_quote_builder.rst applications/sales/sales/sales_quotations/pdf_quote_builder.rst                   # sales/send_quotations/* --> sales/sales_quotations/*
 applications/sales/sales/send_quotations/quote_template.rst applications/sales/sales/sales_quotations/quote_template.rst                         # sales/send_quotations/* --> sales/sales_quotations/*
+
+# applications/voip
+
+applications/productivity/voip/transfer_forward.rst applications/productivity/voip/voip_widget.rst # voip/transfer_forward --> voip/voip_widget
```

Comment:
Please make sure to have a blank line at the end of files (redirect .txt & .rst) :)
```suggestion
applications/productivity/voip/transfer_forward.rst applications/productivity/voip/voip_widget.rst # voip/transfer_forward --> voip/voip_widget

```